nautilus_trader/cache/cache.pyx

```diff
@@ -19,15 +19,13 @@ import time
 import uuid
 from collections import deque
 from decimal import Decimal
-
-from nautilus_trader.cache.config import CacheConfig
-from nautilus_trader.core.rust.model import PriceType as PriceType_py
-
 from cpython.datetime cimport datetime
 from cpython.datetime cimport timedelta
 from libc.stdint cimport uint8_t
 from libc.stdint cimport uint64_t
 
+from nautilus_trader.cache.config import CacheConfig
+from nautilus_trader.core.rust.model import PriceType as PriceType_py
```

Comment:
It's a non-obvious convention, but we standardize pure Python imports above Cython imports for various reasons, so the previous location was intentional.